benchmark_runner/jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py

```diff
@@ -41,16 +66,13 @@ def choose_datetime(self):
         datetime widget
         @return:
         """
+        start_datetime = None
+        end_datetime = None
         start_date_picker = widgets.DatePicker(description='Start Date')
         end_date_picker = widgets.DatePicker(description='End Date')
         start_time_picker = widgets.TimePicker(description='Start Time', value=time(hour=self.DEFAULT_TIME, minute=self.DEFAULT_TIME, second=self.DEFAULT_TIME))
         end_time_picker = widgets.TimePicker(description='End Time', value=time(hour=self.DEFAULT_TIME, minute=self.DEFAULT_TIME, second=self.DEFAULT_TIME))
 
-        display(start_date_picker)
-        display(start_time_picker)
-        display(end_date_picker)
-        display(end_time_picker)
-
         def on_value_change(change):
             global start_datetime
             global end_datetime
```

Comment:
can we declare these two variables in the __init__() method and access using getter methods?

Reply:
@athiruma, Jupyter widgets are working quite different, global variable MUST declare inside on_value_change.
After tr